orbit/goreleaser-linux.yml

```diff
@@ -0,0 +1,36 @@
+project_name: orbit
+
+builds:
+  - id: orbit
+    dir: ./orbit/cmd/orbit/
+    binary: orbit
+    env:
+      # CGO is enabled intentionally for Linux because some users need to be
+      # able to use the cgo versions of the networking libraries (see
+      # https://github.com/fleetdm/fleet/issues/8992)
```

Comment:
Just curious, so we changed this for everyone?

Developers/testers won't be able to test the "real thing" on Linux via [`main.sh`](https://github.com/fleetdm/fleet/blob/main/tools/tuf/test/main.sh) because with CGO you cannot cross compile from macOS to Linux now (am ok with this just something to keep in mind). At some point we may need to move away from goreleaser to have more control of the build process and be able to replicate in developer environments.

Reply:
Is the concern about building Linux releases on macOS? Maybe we can build in a Docker container for that script?

The reason for doing this universally is that it shouldn't affect most deployments, but the few places where they need these changes they can set the environment variables. Then we don't have to start publishing an additional binary type.

Reply:
> Is the concern about building Linux releases on macOS? Maybe we can build in a Docker container for that script?

Yeah you are right, we just need to tweak the dev scripts to use a docker container to build.

> The reason for doing this universally is that it shouldn't affect most deployments, but the few places where they need these changes they can set the environment variables. Then we don't have to start publishing an additional binary type.

Sounds good. Hopefully this change won't have any impact on customers.
Last question, have we tested this new CGO build on Ubuntu/CentOS? Let me know if you need a hand.

Reply:
It was tested in RHEL and (in theory) should work exactly the same on Ubuntu/CentOS (particularly CentOS which is basically RHEL).